src/docs/cursor-ops.dox

```diff
@@ -161,4 +161,52 @@ that may not be modified or freed by the application.  If a longer scope
 is required, the application must make a copy of the memory before the
 cursor is re-used, closed or reset.
 
+@section cursor_bound Apply bounds to the cursor
+
+A cursor may have a set of bounds applied to it, this restricts the key range
+that the cursor operates within.
+
+Suppose we have a table with keys A to Z. We can restrict the cursor to only operate over
+keys L->Q. If the cursor walks past key Q, the cursor will enter an un-positioned state having
+walked off the "end" of the table from it's perspective. Additionally should the cursor perform a
+search_near it will only search within the key range of L->Q. This can provide a performance boost
+when operating within tables that have a large number of deleted or invisible records. The cursor
+can have a lower bound set or an upper bound set, or both.
+
+```
+cursor->set_key(cursor, A)
+cursor->bound(cursor, bound=lower)
+```
+
+Both or either one of the lower and upper bounds can be set, and the unset bound will be assumed to be off the start or
```

Comment:
and the unset bound -> and **any** unset bound

Reply:
[9b21f18](https://github.com/wiredtiger/wiredtiger/pull/8367/commits/9b21f186bb973ba5c89e96925bf8becfecf64b56)